app/src/controllers/newsfeed/poster/poster.ts

```diff
@@ -57,6 +58,15 @@ export class Poster {
     }
   }
 
+  ngOnInit(){
+    var self = this;
+    var gun = self.gun;
+    gun.read('draft.status', function(message){
```

Comment:
Please, use fat arrow functions (like `(message) => { }`) and don't use `self = this` (not needed in TypeScript because of fat arrow functions)